scout/adapter/mongo/variant_loader.py

```diff
@@ -569,13 +569,17 @@ def _is_causative_other_cases(
         clinical_variant = "".join([variant_prefix, "_clinical"])
         research_variant = "".join([variant_prefix, "_research"])
 
-        var_causative_events_count = self.event_collection.find(
-            {
-                "verb": {"$in": ["mark_causative", "mark_partial_causative"]},
-                "category": "variant",
-                "subject": {"$in": [clinical_variant, research_variant]},
-            }
-        ).count()
```

Comment:
My bad - this has been deprecated like for ever.

Reply:
Wow, haven't seen one of those in a long time. Well spotted!